modules/navier_stokes/test/tests/postprocessors/gold/diverging_INSFV_quad_xy_noslip.csv

```diff
@@ -1,3 +1,3 @@
 time,inlet_advected_energy,inlet_mass_constant,inlet_mass_matprop,inlet_mass_variable,inlet_momentum_x,inlet_momentum_y,mid1_advected_energy,mid1_mass,mid1_momentum_x,mid1_momentum_y,mid2_advected_energy,mid2_mass,mid2_momentum_x,mid2_momentum_y,outlet_advected_energy,outlet_mass,outlet_momentum_x,outlet_momentum_y
 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-1,-303.22631741677,-1,-1,-1,0,-1,310.91762190643,1,-0.00049524449285384,1.1835461195569,310.1724016237,1,0.055136233385212,0.92892382096507,310.03798983956,1,0.015667474006117,0.83548919055089
```

Comment:
@lindsayad What do you think of these changes?

Reply:
These changes come from another PR. They're his commits. They're just added to make this pass. It'll go in before this.